src/main/java/io/scif/formats/JavaFormat.java

```diff
@@ -92,7 +86,7 @@ public void setDest(final RandomAccessOutputStream stream,
 			IOException
 		{
 			super.setDest(stream, imageIndex, config);
-			if (getStream().length() == 0) writeHeader();
+			if (getStream().length() == 0) { writeHeader(); }
```

Comment:
This is not ImageJ style. When curly braces are present, the opening brace goes on the same line, the statement(s) go on following line(s), and closing brace on its own line. The Eclipse code style formatter does this.